test/outside/off/off_shops_manager_test.dart

```diff
@@ -114,6 +115,16 @@ void main() {
     expect(shopsResult.isErr, isTrue);
   });
 
+  test('do not fetch shops when country not in enabled list', () async {
+    await _initShops(
+      cameraPos: Coord(lat: 10, lon: 10),
+      addressOfAnyCoords: Ok(OsmAddress((e) => e.countryCode = RUSSIA)),
+      offApiShops: Ok(someOffShops),
+    );
+    final shopsResult = await offShopsManager.fetchOffShops();
+    expect(shopsResult.isErr, isTrue);
```

Comment:
Hmmm...
Maybe we should rather return an empty collection here?
If a country is not supported yet, it's not an error per se.